scripts/editor/props.js

```diff
@@ -25,18 +25,18 @@ export const props = (attributes, realName, newName = '', isBlock = false, names
 	// Get global window data.
 	const globalData = window['eightshift'][namespace].dependency;
 
-	// If component use components dependency tree.
-	// Add the current component name to the dependency array.
-	let dependency = [
-		...globalData.components[realName],
-		newNameInternal
-	];
+	let dependency = [];
 
-	// If block use blocks dependency tree.
+	// If block use blocks dependency tree, if component use components dependency tree.
```

Comment:
```suggestion
	// If dealing with a block, use the block's dependency tree, if dealing with a component use the component's dependency tree.
```

Reply:
I mean, the original version is more concise and relays the same info 😄